grails-app/services/au/org/emii/portal/BulkDownloadService.groovy

```diff
@@ -145,4 +145,14 @@ class BulkDownloadService {
         zipStream.write bytes, 0, bytes.length
         zipStream.closeEntry()
     }
+
+    public static void copy(final InputStream input, final OutputStream output, final int bufferSize) throws IOException
```

Comment:
I actually wanted to open a bug on that. I don't understand why the behaviour is not compatible with `org.apache.commons.io`. It's silly.